lib/rsync.js

```diff
@@ -28,6 +28,8 @@ module.exports = rsync  =  {
   runRsync: (config, machine, callback) => {
     let logger = log4js.getLogger(machine.name);
     let backupDate = new Date(Date.now());
+    machine.backupStartTime=new Date().getTime();
+    machine.inProgress=true;
```

Comment:
It would be smarter / easier to maintain if we also set this flag to false within the `runRsync` call. It would have to happen when either there is a success or a failure, but I think we want to be robust enough to handle all potential error occurrences in this call anyway. 

Despite this I think we can merge in this code for now. Just something to keep in mind for the future.